Terminal.Gui/Views/ListView.cs

```diff
@@ -264,6 +271,11 @@ public IListDataSource Source
             }
             _source = value;
 
+            if (_source is { })
```

Comment:
This has the potential to cause multiple subscriptions.

Reply:
It's only check for null value here. It only subscribe if not null and it's unsubscribe the previous.

Reply:
That's a check for not null, not a check for null.

Reply:
I said wrongly for null but I meant for not null. You are right. I'm unsubscribing on the Dispose method and I need to do it here before subscribe the new one.

Reply:
I think I fixed this with the 893a6c8c322816e72d7d3093ebc8197d7c15574a commit with unit tests that is working as expected. Only the `ListWrapper` managed outside the `ListView` the user is responsible to dispose it which will unsubscribe the event. Thanks for to alert me with this.